packages/cli/src/cli/commands/init.test.ts

```diff
@@ -0,0 +1,128 @@
+import fs, { readFileSync } from "fs";
+import path from "path";
+import tmp from "tmp";
```

Comment:
Seems like this package needs to be added to the package.json? I get this "Cannot find module" error locally

![Screenshot 2023-08-29 at 11 47 34 AM](https://github.com/pantheon-systems/pantheon-content-cloud-sdk/assets/87580113/c4e6c087-2f17-4c05-957a-1081074de044)

Reply:
I think its already present as a indirect dependency. Since tests are also passing on the Github actions.
Anyways I will add it as a direct dependency.